src/main/data/languages.json

```diff
@@ -440,6 +440,14 @@
       "text"
     ]
   },
+  {
+    "dcncLanguage": "Macedonian",
+    "dcncTag": "MK",
+    "rfc5646Tag": "mk",
+    "use": [
+      "audio"
```

Comment:
@SteveLLamb Can you confirm that this language cannot be used with subs?

Reply:
Found an internal example of it being used for subs, and no where stated it cannot. Updated via c5197c5